docs/CONTRIBUTING.md

```diff
@@ -16,16 +16,18 @@
 
 1. Install Node.js and `npm` on your development machine by following the instructions on the Node.js website.
 2. Clone the project repository from GitHub.
-3. Open the project in your preferred IDE or code editor.
+3. Open the `frontend` folder in your preferred IDE or code editor.
 4. Install the required dependencies by running the **`npm install`** command in your terminal.
-5. Make changes to the code as needed, ensuring that you follow the project's coding standards and best practices.
-6. Next we will create a `.env` file, which contains the following key-value pairs:
+5. Next we will create a `.env` file, which contains the following key-value pairs:
```

Comment:
See [here](https://github.com/AmanNegi/FreshNest/tree/main/frontend/react#readme), how it mentions we can use remote as well as local URL.